filer/admin/folderadmin.py

```diff
@@ -271,11 +273,13 @@ def directory_listing(self, request, folder_id=None, viewtype=None):
 
         list_type = get_directory_listing_type(request) or settings.FILER_FOLDER_ADMIN_DEFAULT_LIST_TYPE
         if list_type == TABLE_LIST_TYPE:
-            size = "40x40"  # Prefetch thumbnails for listing
-            size_x2 = "80x80"
+            s = FILER_TABLE_ICON_SIZE  # Prefetch thumbnails for table view
```

Comment:
Can we please more descriptive variable name than just `s` here?

Reply:
Yeah, I fully removed it because the descriptive names are actually the settings like `FILE_TABLE_ICON_SIZE`.